### PR TITLE
Adding a note re unit test.

### PR DESCRIPTION
**Pull request by achuzhoy**
We now require ading unit test for every package.
Adding a respective note to the readme.

### CODE REVIEW

**Review by kononovn — DISMISSED** [review]
/lgtm

File: README.md [hunk 1]
```diff
@@ -111,6 +111,7 @@ The project uses a development method - forking workflow
    the [eco-goinfra](https://github.com/openshift-kni/eco-goinfra).
 8) The pull request gets approved from at least 2 reviewers for merge and is merged into
    the [eco-goinfra](https://github.com/openshift-kni/eco-goinfra) .
+#### Note: Every new package requires a coverage of <ins>ALL</ins> its public functions with unit tests. Examples of unit tests can be found in the repo and are located in the same folder with the package they cover and named after the CRD, for example [configmap_test.go](./pkg/configmap/configmap_test.go) or [metallb_test.go](./pkg/metallb/metallb_test.go).
```
Comment (trewest) [inline README.md:114]:
```suggestion
#### Note: Every new package requires a coverage of <ins>ALL</ins> its public functions with unit tests. Unit tests are located in the same package as the resource under test in a file with the name *<resource>*_test.go. Examples can be found in [configmap_test.go](./pkg/configmap/configmap_test.go) and [metallb_test.go](./pkg/metallb/metallb_test.go).
```

**Review by trewest — APPROVED** [review]
/lgtm

**Review by kononovn — APPROVED** [review]
/lgtm

**Pull request merged**